code/game/atoms_movable.dm

```diff
@@ -440,7 +440,7 @@
 	if(oldarea != newarea)
 		newarea.Entered(src, direction)
 
-	Moved(oldloc, direction)
+	Moved(oldloc, direction, FALSE, old_locs)
```

Comment:
Can you put named arguments on this?

Reply:
The answer is "yes, but named arguments cost more than a normal call"; and this is a very hot proc.